source/releasenotes/2024-08-19-FS-directory-changes.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Preventing filesystem overload to ensure site stability"
+published_date: "2024-09-19"
+---
+Starting September 19, 2024, the Pantheon platform will automatically reject file writes (beyond the 100k limit) that could push the filesystem beyond its capacity. This precaution helps avoid catastrophic failures, ensuring the site remains online and operational.
```

Comment:
```suggestion
Writing more than 100,000 files to the same directory can push the Pantheon filesystem beyond its intended design. Starting September 19, 2024, our platform will automatically reject file writes if the directory is at capacity. This precaution helps avoid catastrophic failures to ensure the site remains online and operational.
```

Reply:
updated wording to bring more clarity around "writes" vs "files", and writing out 100k to 100,000 since this could be confused with 100kb too.